lib/SILOptimizer/Mandatory/TFConstExpr.cpp

```diff
@@ -335,6 +334,19 @@ SymbolicValue ConstExprFunctionState::computeConstantValue(SILValue value) {
                                                evaluator.getAllocator());
   }
 
+  // This one returns the address of its enum payload.
```

Comment:
ConstExprFunctionState::computeConstantValue() is getting very long. one option is to move all address related insts into a separate helper func.

Reply:
Agreed, another option is to split it up using the visitor pattern.  I'll see what makes sense.

Thank you for the review!